README.md

```diff
@@ -1,2 +1,28 @@
 # jest-evaluator-action
-Jest evaluator Action for Tryber projects
+Jest evaluator action for Tryber projects
+
+This action evaluate Tryber projects with [Jest](https://jestjs.io/) library.
+
+## Inputs
+
+## Outputs
+
+### `result`
+
+Jest unit tests JSON results in base64 format.
+
+## Simple usage example
+```yml
+uses: betrybe/jest-evaluator-action
+```
+
+## How to get result output
+```yml
```

Comment:
It's not clear to me where and why this `yml` will be used. Maybe it's a good idea to describe it better in this topic.

Reply:
I'll add a Github link about GitHub Actions. I think this is the better way to learn about GitHub Actions